data/json/items/generic.json

```
@@ -2400,6 +2400,15 @@
     "flags": [ "SLEEP_AID" ],
     "category": "other"
   },
+  {
+    "type": "GENERIC",
+    "id": "bodypillow_makeshift",
+    "copy-from": "bodypillow",
+    "name": { "str": "makeshift body pillow", "str_pl": "makeshift body pillows" },
```

```suggestion
    "name": { "str": "makeshift body pillow" },
```
The plural can be automatically generated because it is only the singular string + "s".